install/kubernetes/helm/istio/values-istio-one-namespace-trust-domain.yaml

```diff
@@ -14,5 +14,4 @@ global:
 
   trustDomain: test.local
 
-istiotesting:
   oneNameSpace: true
```

Comment:
Shouldn't you remove this line too?

Reply:
We should use this one: https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/values.yaml#L278

Reply:
Ok, that's why my search couldn't find it..